frontend/src/components/pages/RegistrantExperience/RegistrationSteps/ReviewRegistration/PaymentSummary.tsx

```diff
@@ -96,7 +94,7 @@ const PaymentSummary = ({
           <Tbody>
             {items.map((item, index) => (
               <Tr
-                key={item.name}
+                key={index}
```

Comment:
can we make keys more descriptive? There is a greater chance of overlapping keys if we keep making them numbers